openapi_python_client/parser/properties/union.py

```diff
@@ -10,7 +10,14 @@
 from ...utils import PythonIdentifier
 from ..errors import ParseError, PropertyError
 from .protocol import PropertyProtocol, Value
-from .schemas import Schemas
+from .schemas import Schemas, get_reference_simple_name, parse_reference_path
+
```

Comment:
This file defines `UnionProperty`, which is the generator's internal class for representing any kind of `anyOf` or `oneOf` situation.

When parsing a schema, the generator will call `UnionProperty.build` (line 38), which returns a tuple of 1. either a valid `UnionProperty` or an error, and 2. an updated version of the `Schemas` object that keeps track of all parsed schemas so far. This is a standard pattern used with all of the generator's `___Property` types.